### ISSUE TITLE
Getting lose of group ownership

### ISSUE THREAD

**Issue by ruKurz** [RocketChat/Rocket.Chat#13247]
## Description
If the last owner of a channel has been locked, there is no way to create a new owner for that channel.

## Steps to reproduce
1. Create a mew private group being logged in with user X
2. Logout and login with an Admin user
3. Try to add a new owner without using the login of user X

## Environment
RC: 0.71.1
Platform: linux
Architektur: x64
Node-Version: v8.11.4

## Background
When someone leaves a project, a group, a company, whatever (and his Rocket.chat user is deleted or disabled), the administrator needs the authority to assign a new owner for the groups and channels, whose last owner is now the more available person.

**Comment by reetp** [comment]
Hi. I think this may have been seen before. Please have a thorough search in Issues and close this if you find a duplicate.

Also when reporting Issues always try on latest as it may have been fixed as Rocket development is still fast. Latest us currently 0.73.2 and there have been a lot of fixes since your version. Please reas the changelogs for more.

**Comment by ruKurz** [comment]
@reetp you're totally right by advising me to first search in already reported issues as well as pointing me using latest and greatest.

I'm involved in "Enterprise Business" therefore it is quite hard spending much time in reporting issues. I know this is a bad excuse. Anyway feedback is more valuable to an project compared with no feedback ;-)

I will scan the existing issues and close this if duplicated.

**Comment by ruKurz** [comment]
This issue may be related with:
- https://github.com/RocketChat/Rocket.Chat/issues/12978
- https://github.com/RocketChat/Rocket.Chat/issues/11941

**Comment by mrsimpson** [comment]
as an admin, you can assign the owner permission with respect to a room (private or not) using the permissions-administration:
Open the permissions-table, select "owner". Now, you can pick any room and edit the users with this permission

<img width="874" alt="Bildschirmfoto 2020-02-14 um 22 21 52" src="https://user-images.githubusercontent.com/17176678/74568554-88eb1900-4f78-11ea-938e-57f29644ca50.png">
